jsconfig.json

```diff
@@ -6,6 +6,7 @@
       "@images/*": ["./public/assets/images/*"],
       "@public/*": ["./public/assets/*"],
       "@components": ["./src/app/_components/index"],
+      "@components/*": ["./src/app/_components/*"],
```

Comment:
🟡 to more strictly enforce direct component import from specific file or module (like @components/Modal), and not just @components, consider removing "@components" from jsconfig altogether. In that case though, imports from @components in other files should be fixed as well

Reply:
done, thanks